webservice/app/Transformers/UserTransformer.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Transformers;
+
+use App\User;
+use League\Fractal\TransformerAbstract;
+
+class UserTransformer extends TransformerAbstract
+{
+    /**
+     * Transform a user.
+     *
+     * @param  User $user
+     *
+     * @return array
+     */
+    public function transform(User $user)
+    {
+        return [
+            'id' => $user->id,
+            'name' => $user->name,
+            'email' => $user->email,
+            'created_at' => $user->created_at->format('Y-m-d H:i:s'),
```

Comment:
is new for me too.
I already knew that there was a recommended format, I was just validating which was